e2e/result-cache-1/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+function test_result_cache1() {
+    echo -n > phpstan-baseline.neon
+    ../../bin/phpstan -vvv
+    patch -b src/Bar.php < patch-1.patch
+    cat baseline-1.neon > phpstan-baseline.neon
+    ../../bin/phpstan -vvv
+    mv src/Bar.php.orig src/Bar.php
+    echo -n > phpstan-baseline.neon
+    ../../bin/phpstan -vvv
+} 2>&1
```

Comment:
I need to redirect stderr to stdout for all commands of the test because bashunit reports a test-failure as soon as something is written to stdout

Reply:
I think I want to ask the authors what's the best way to do it here.